test/tap/tests/test_flush_logs-t.sh

```
@@ -99,14 +115,53 @@ fn_get_rotations () {
+	# failure. By polling we convert a timing race into a max-latency
+	# bound - on a fast machine the test is as quick as before, on a
+	# slow one it waits up to max_seconds instead of failing.
+	local target=${1}
+	local max_seconds=${2:-30}
+	local i=0
+	local count=0
+	local max_iterations=$(( max_seconds * 2 ))
+	while [ $i -lt $max_iterations ]; do
+		count=$(fn_get_rotations)
+		if [ "$count" -ge "$target" ]; then
+			echo $count
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)